lib/sync_checker/checks/translations_check.rb

```diff
@@ -0,0 +1,32 @@
+module SyncChecker
+  module Checks
+    TranslationsCheck = Struct.new(:expected_locales) do
+      attr_reader :content_item
+      def call(response)
+        stringified_locales = expected_locales.map(&:to_s)
```

Comment:
When you're calling this from `EditionBase`, it's possible for `expected_locales` to be `nil`. In that case, can we return early here?